components/micro-gateway-core/src/main/ballerina/gateway/filters/subscription_filter.bal

```diff
@@ -82,6 +83,25 @@ public type SubscriptionFilter object {
                                     return true;
                                 }
                             }
+                            }
+                            else
+                            {
+                                authenticationContext.authenticated = true;
+                                authenticationContext.tier = "Unlimited";
+                                authenticationContext.apiKey = jwtToken;
+                                authenticationContext.applicationId = "";
+                                authenticationContext.applicationName = "";
+                                authenticationContext.applicationTier = "Unlimited";
+                                authenticationContext.subscriber = "";
+                                authenticationContext.consumerKey = decodedPayload.consumerKey.toString();
+                                authenticationContext.apiTier = decodedPayload.apiTier.toString();
+                                authenticationContext.apiPublisher = "";
+                                authenticationContext.subscriberTenantDomain = "";
+                                runtime:getInvocationContext().attributes[KEY_TYPE_ATTR] = authenticationContext.keyType
+                                ;
```

Comment:
fix formatting issue

Reply:
this code has revamped and formatted with 4d34de4